person.rb

```diff
@@ -18,6 +25,31 @@ def can_use_services?
   end
 
   private :of_age?
+
+  def correct_name
+    @name
+  end
 end
 
-Person.new(14, 'Jonathan')
+class BaseDecorator < Nameable
+  def initialize(nameable)
+    super
+    @nameable = nameable
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/216043408-c428d580-107c-4d04-bfe2-473c5e8e0467.png)

- [ ] When I try to capitalize or trim a person then I am facing this error which you can see in the image above :arrow_up: Kindly fix this error by adding parentheses after `super` such as `super()` so this error won't come :+1: